app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,5 @@
+class PostsContoller < ApplicationController
```

Comment:
- Hey, we've made a typo here as it should be `PostsController`. Please, kindly update this to fix the error in the routes.